packages/ui-library/src/elements/toaster/index.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable complexity */
+import { Transition } from "@headlessui/react";
+import { XIcon } from "@heroicons/react/outline";
+import classNames from "classnames";
+import { isArray, noop } from "lodash";
+import PropTypes from "prop-types";
+import React, { createContext, useCallback, useContext, useEffect } from "react";
+
+const ToasterContext = createContext( { handleDismiss: noop } );
+
+export const toasterClassNameMap = {
+	position: {
+		"bottom-center": "yst-translate-y-full",
+		"bottom-left": "yst-translate-y-full",
+		"top-center": "yst--translate-y-full",
+	},
+};
```

Comment:
Is this correct? I see it's copied from the Notification component so I guess it is 😅 Also not sure about the double hyphens in `yst--translate-y-full`.

Reply:
Yeah, that fooled me too! 😓 
It is correct, the rationale is: if your toaster is placed bottom center or bottom left, you want a positive y translation (considering the 0 of the y axis to be at the bottom of the corner); for a top center toaster, you want to invert the transition (from bottom to top), hence the `-` in front of the translation 🙂

Reply:
Oh haha wow 🤯 Thanks! 😄